internal/migrations/migrations.go

```diff
@@ -14,6 +14,7 @@ func MigratePre(db *gorm.DB) error {
 func pre() []*gormigrate.Migration {
 	preMigrations := []*gormigrate.Migration{
 		populateInfraEnv(),
+		modifyEventsId(),
```

Comment:
Don't put things in preMigrations unless there is a _very_ good reason. Ideally this wouldn't exist, so if you do need to put this migration here, please add a full explanation to the commit message and PR description.

See: https://github.com/openshift/assisted-service/blob/master/docs/dev/migrations.md#pre-migrations

Reply:
The id has to be migrated before AutoMigrate is being run.  Otherwise, AutoMigrate fails

Reply:
This is important, can you put this in the PR description and commit message?

Reply:
Also I guess this is the only table we have with an integer pk column?

Reply:
Saw that it was added :+1: